prettyPrinters/lldb_formatters/__init__.py

```diff
@@ -0,0 +1,66 @@
+from rust_types import TYPE_TO_REGEX
+from rust_types import RustType
+
+# noinspection PyUnresolvedReferences
+import lldb_formatters.lldb_providers
+from lldb_formatters.lldb_providers import *
+
+
+# noinspection DuplicatedCode
+def __lldb_init_module(debugger, _dict):
+    def register_providers(rust_type, summary=None, synth=None):
+        regex = TYPE_TO_REGEX[rust_type].pattern
+
+        if summary:
+            func_name = summary.__name__
+            debugger.HandleCommand(
+                'type summary add -F lldb_formatters.lldb_providers.{} -e -x -h "{}" --category Rust'.format(func_name,
+                                                                                                             regex)
+            )
+        if synth:
+            class_name = synth.__name__
+            debugger.HandleCommand(
+                'type synthetic add -l lldb_formatters.lldb_providers.{} -x "{}" --category Rust'.format(class_name,
+                                                                                                         regex)
+            )
+
+    register_providers(RustType.STR, StdStrSummaryProvider)
```

Comment:
nit: maybe it's worth to pass providers with name argument syntax to increase readability?